rules/S7714/csharp/metadata.json

```
@@ -0,0 +1,53 @@
+{
+  "title": "XSLT Transformations should not be vulnerable to injection attacks",
+  "type": "VULNERABILITY",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "45min"
+  },
+  "tags": [
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-7714",
+  "sqKey": "S7714",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "HIGH",
+      "RELIABILITY": "MEDIUM",
+      "SECURITY": "LOW"
+    },
+    "attribute": "CONVENTIONAL"
+  },
```

We usually have that in our rules
```suggestion
"code": {
    "impacts": {
      "SECURITY": "BLOCKER"
    },
    "attribute": "COMPLETE"
  },
```